src/ipvs/ip_vs_conn.c

```diff
@@ -495,10 +542,7 @@ static int conn_expire(void *priv)
     /* refcnt == 1 means we are the only referer.
      * no one is using the conn and it's timed out. */
     if (rte_atomic32_read(&conn->refcnt) == 1) {
-        if (conn->flags & DPVS_CONN_F_TEMPLATE)
-            dpvs_timer_cancel_nolock(&conn->timer, true);
-        else
-            dpvs_timer_cancel_nolock(&conn->timer, false);
+        dp_vs_conn_detach_timer(conn);
```

Comment:
"-nolock" timer api should be used in timer's callback.

Reply:
Done.